protocol/grpc/client.go

```diff
@@ -39,9 +44,24 @@ type Client struct {
 
 // NewClient ...
 func NewClient(url common.URL) *Client {
-	conn, err := grpc.Dial(url.Location, grpc.WithInsecure(), grpc.WithBlock())
-	if err != nil {
-		panic(err)
+	var (
+		conn   *grpc.ClientConn
+		err    error
+		tracer opentracing.Tracer
+	)
+	// if global trace instance was set in filter, it means trace function enabled
+	if opentracing.IsGlobalTracerRegistered() {
```

Comment:
You don't need to check this because if the GlobalTracer is not set, the opentracing will do nothing and return a mockTracer.

Reply:
hi, if GlobalTracer not set, it means the tracer function not enabled in dubbo-go Grpc，so i think it is necessary to check this first